src/Java.Interop/Java.Interop.targets

```diff
@@ -14,13 +14,13 @@
   <Target Name="BuildJniEnvironment_g_cs"
       BeforeTargets="BeforeCompile"
       Inputs="$(_JNIEnvGenPath)"
-      Outputs="Java.Interop/JniEnvironment.g.cs;$(IntermediateOutputPath)jni.c">
+      Outputs="$(IntermediateOutputPath)JniEnvironment.g.cs;$(IntermediateOutputPath)jni.c">
     <MakeDir Directories="$(IntermediateOutputPath)" />
     <Exec
-        Command="$(_RunJNIEnvGen) Java.Interop/JniEnvironment.g.cs $(IntermediateOutputPath)jni.c"
+        Command="$(_RunJNIEnvGen) $(IntermediateOutputPath)JniEnvironment.g.cs $(IntermediateOutputPath)jni.c"
     />
     <ItemGroup>
-      <Compile Include="$([System.IO.Path]::Combine('Java.Interop','JniEnvironment.g.cs'))" KeepDuplicates="false" />
+      <Compile Include="$(IntermediateOutputPath)JniEnvironment.g.cs" KeepDuplicates="false" />
```

Comment:
Reading https://github.com/xamarin/java.interop/commit/99f89905239b66577ba289099a6909af6064dd3c, we might not need the `%(KeepDuplicates)` anymore. I don't think the default C# wildcards would include it.

But we could also just leave this for now.